gatewayconfig/bluetooth/characteristics/add_gateway_characteristic.py

```diff
@@ -40,7 +65,7 @@ def StartNotify(self):
 
         self.PropertiesChanged(constants.GATT_CHRC_IFACE, {"Value": self.notifyValue},
                                [])
-        self.add_timeout(30000, self.AddGatewayCallback)
+        # self.add_timeout(30000, self.AddGatewayCallback)
```

Comment:
Do we need to leave the commented code in?

Reply:
removed.